src/RestClient/Authenticators/OAuth.AspNetCore/src/HttpContextTokenStore.cs

```diff
@@ -109,9 +135,9 @@ private Token CreateToken(TokenType tokenType, string value, DateTimeOffset? exp
             };
         }
 
-        private static async Task<DateTimeOffset?> GetTokenExpireTimeAsync(HttpContext httpContext)
+        private static DateTimeOffset? GetTokenExpireTimeAsync(AuthenticationProperties properties)
```

Comment:
remove the `Async` suffix as it is not Task anymore?

Reply:
ooh good catch ty done